business_requirement/models/business.py

```diff
@@ -275,6 +275,22 @@ def name_get(self):
             result.append((br.id, formatted_name))
         return result
 
+    @api.model
+    def name_search(self, name, args=None, operator='ilike', limit=100):
+        """
+        Search BR based on Name or Description
+        """
+        # Make a search with default criteria
+        names1 = super(BusinessRequirement, self).name_search(
```

Comment:
@sudhir-serpentcs Since a while I haven't see a variable with numbers.

think this could named `names`
second one `descriptions`

@elicoidal ?